### ISSUE TITLE
The speed at which some objects return to their position seems unusually fast

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/number-compare#22]
**Test device**
MacBook Air M1 chip

**Operating System**
13.2.1

**Browser**
Safari 

**Problem description**
For https://github.com/phetsims/qa/issues/917, on the Lab Screen:
If I grab a card (number, symbol or ten card) and let go, the speed that it returns to its original location seems to be much faster than I am used to seeing in sims. It is also faster than when an object, such as a dog, moves back to its toolbox. 

**Steps to reproduce**
1. On the lab screen, grab the 1 card and move it over the 10 card
2. Let go

**Visuals**

https://user-images.githubusercontent.com/87318828/225326380-d2b8c014-7129-4adf-b489-55cf8b22b56d.mp4

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Number Compare‬
URL: https://phet-dev.colorado.edu/html/number-compare/1.0.0-dev.6/phet/number-compare_all_phet.html
Version: 1.0.0-dev.6 2023-03-15 01:30:22 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.3 Safari/605.1.15
Language: en-US
Window: 1203x654
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}
</details>

**Comment by chrisklus** [comment]
Nice catch @Nancy-Salpepi! Agreed the animation should be slowed. How this is issue is fixed will partially depend on decisions made in https://github.com/phetsims/number-compare/issues/24. I will also remember to tweak the animation speed of the ten frames. Marking on hold until https://github.com/phetsims/number-compare/issues/24 gets feedback.

**Comment by amanda-phet** [comment]
Discussed with @chrisklus . 

This should be improved with #24 . Let's check it after that is updated, but I'm inclined to leave this as-is because the number cards are using Fractions code. It's a really wide carousel, so an object that must fly really far will fly faster, but I think it will not be too bad after #24. It also is probably not worth the hassle to make this change. 

We could make the change to the ten frames and comparison symbols, though, since that is Number Suite Common code, but I didn't think it seemed too fast. However I'm fine if we want to change those to match the speed of the objects.

**Comment by amanda-phet** [comment]
Update: looks like it _is_ number suite common code, and @chrisklus can change it. 

@chrisklus will change all of these animations to be constant speed.

**Comment by zepumph** [comment]
https://github.com/phetsims/number-compare/issues/24 is closed, unholding!

**Comment by chrisklus** [comment]
Changes committed above to match all of the animation styles. @zepumph and I have separately verified on our own machines that this is working since @Nancy-Salpepi is out for the remainder of the day and we would like to open a NP dev test! Closing.
